entry_types/scrolled/package/src/contentElements/videoEmbed/VideoEmbed.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactPlayer from 'react-player';
+import styles from './VideoEmbed.module.css';
+import classNames from 'classnames';
+import {InlineCaption} from "../inlineImage/InlineCaption";
+
+export function VideoEmbed({configuration}) {
+  return (
+    <div className={styles.VideoEmbed}>
+      <div className={classNames(styles.embedWrapper, styles[configuration.aspectRatio])}>
+        <ReactPlayer className={styles.embedPlayer}
+                     key={btoa(JSON.stringify(configuration))}
```

Comment:
Why is this needed? If it is needed, I'd suggest constructing a less opaque string from the content element id etc.

Reply:
Changing the url reloads the player component but changing options doesn't (by default), so `hideControls` doesn't get picked up.
I took the idea from [here](https://github.com/CookPete/react-player/issues/209).
Since the element only has two options currently I could construct the key from those, but when adding new options later one would certainly forget to add them to the key, that's why I constructed the key from the whole configuration object to make sure the key changes when options change.